plugins/outputs/groundwork/log_adapter.go

```diff
@@ -0,0 +1,87 @@
+package groundwork
+
+import (
+	"context"
+	"encoding/json"
+	"log/slog"
```

Comment:
Use

```suggestion
	"log/slog" //nolint:depguard // Required for wrapping internal logging facility
```

to silence the linter warning.

Reply:
done